optuna/visualization.py

```diff
@@ -92,6 +99,164 @@ def _get_intermediate_plot(study):
     return figure
 
 
+def plot_contour(study, params=[]):
```

Comment:
I'd expect an empty figure, if I specified `params=[]`. How about using `None` instead?

Reply:
Exactly.
However, other PR #540 and #531 have similar implementations, so after everything has been merged, create a new PR about this part.